content/medo_manual/inloggen_en_veiligheid.md

```diff
@@ -0,0 +1,42 @@
+---
+title: Inloggen en veiligheid
+sort: 100
+---
+
+* TOC
+{:toc}
+
+## Eerste gebruik
```

Comment:
Misschien ook wel aardig om hier straks naar te linken in de uitnodigings e-mail